params/network_config_test.go

```diff
@@ -1,25 +1,23 @@
-package params_test
```

Comment:
one doesn't open a `+4 -6` PR expecting a big discussion, but here I go:

What's wrong about having tests in a separate package? I normally try to do it this way. 
It forces you to test behavior/interface and not implementation of the target package, it prevents accidental exposure of test stuff into the real app. I normally will try to

Reply:
TIL 🙌 

Apparently this is a known (just not to me) approach in Go, which indeed forces you to test public API.
I'll close the PR, thank you @dlipicar!